Code/GraphMol/ChemReactions/Wrap/Enumerate.cpp

```diff
@@ -293,8 +294,7 @@ library2.InitFromString(s)\n\
 for result in itertools.islice(libary2, 1000):\n\
     # do something with the next 1000 samples\n\
 ";
-    python::class_<EnumerateLibraryWrap,
-                   EnumerateLibraryWrap*,EnumerateLibraryWrap&,
+    python::class_<EnumerateLibraryWrap, boost::noncopyable,
```

Comment:
curiosity question: why is the `boost::noncopyable` needed here?

Reply:
I was just making changes until it worked.  It wasn’t the virtual destructor which was the obvious thing to fix.  I think it was the ref and pointer definitions in the boost python  class definition that were causing the issue.   I’ll try a run without noncopyable.